database/migrations/20210531184352-addCaptureTakenAtColumn.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
I would add these migration scripts as part of the project that owns the schema. For e.g. `raw_capture_feature` is a schema owned by the https://github.com/Greenstand/webmap-query-service-consumer project.

Sorry if i wasn't clear about it.

Reply:
Also, this column addition should be a separate PR associated with an issue in that project that ensures the webmap-query-service-consumer  has associated code is added if needed. Here is that ticket https://github.com/Greenstand/webmap-query-service-consumer/issues/14